libs/sdk-ui-web-components/src/visualizations/components.shadow.css

```diff
@@ -0,0 +1,13 @@
+@import "~@gooddata/sdk-ui-filters/styles/css/main.css";
+@import "~@gooddata/sdk-ui-charts/styles/css/main.css";
+@import "~@gooddata/sdk-ui-geo/styles/css/main.css";
+@import "~@gooddata/sdk-ui-pivot/styles/css/main.css";
+@import "~@gooddata/sdk-ui-kit/styles/css/main.css";
+@import "~@gooddata/sdk-ui-ext/styles/css/main.css";
+@import "~@gooddata/sdk-ui-dashboard/styles/css/main.css";
```

Comment:
Ideally, we would want to have 2 sets of styles - for insight and for dashboard. This is a compromise here to keep the setup simple. I tried a bunch of different style loading options and this is the optimal solution. To make it completely clean, would likely need to create a custom WebPack loader or plugin - too much work for the benefit.

Anyway, this styles are extracted to a separate highly cachable file and injected into Shadow DOMs with a link tag, so extra dashboard styles should not make a big difference even if user is only using `gd-insight`.